ee/clickhouse/queries/test/__snapshots__/test_paths.ambr

```diff
@@ -965,20 +965,33 @@
                      groupArray(path_item) as paths
               FROM
                 (SELECT e.timestamp AS timestamp,
-                        e.person_id AS person_id,
+                        pdi.person_id AS person_id,
                         if(e.event = '$screen', replaceRegexpAll(JSONExtractRaw(properties, '$screen_name'), '^"|"$', ''), if(e.event = '$pageview', if(length(replaceRegexpAll(JSONExtractRaw(properties, '$current_url'), '^"|"$', '')) > 1, replaceRegexpAll(replaceRegexpAll(JSONExtractRaw(properties, '$current_url'), '^"|"$', ''), '/$', ''), replaceRegexpAll(JSONExtractRaw(properties, '$current_url'), '^"|"$', '')), e.event)) AS path_item_ungrouped,
                         multiMatchAnyIndex(path_item_ungrouped, NULL) AS group_index,
                         if(group_index > 0, NULL[group_index], path_item_ungrouped) AS path_item
                  FROM events e
+                 INNER JOIN
```

Comment:
Ideally this join doesn't come back?

Reply:
It doesn't.

Not sure how snapshot updating works with the test split, but this snapshot is now coming from non-person-on-events tests.